core/dbt/cli/main.py

```diff
@@ -73,6 +75,13 @@ def cli(ctx, **kwargs):
     if flags.RECORD_TIMING_INFO:
         ctx.with_resource(profiler(enable=True, outfile=flags.RECORD_TIMING_INFO))
 
+    cli_vars = parse_cli_vars(getattr(flags, "vars", "{}"))
+
+    # TODO need profile to exisit
+    profile = None
+    # project need profile to render because it requires knowing Target
+    ctx.obj["project"] = load_project(flags.PROJECT_DIR, profile, cli_vars)
```

Comment:
Do we need to initialize ctx.obj to an empty dictionary prior to the assignment here? Testing this out locally, I get the following error: 

```    
   ctx.obj["project"] = load_project(project_root, profile, cli_vars)
TypeError: 'NoneType' object does not support item assignment
```

Reply:
And we talked about testing this file during standup today, @iknox-fa will add some tests to test calls to the new CLI interface will run without error. Currently not possible due to profile doesn't exist